packages/react-app/src/pages/_app.tsx

```diff
@@ -14,10 +14,46 @@ import '../styles/css/nprogress.css';
 import { CustomerContext, getCustomerFromBountyId, setCustomerFromLocalStorage } from '../context/CustomerContext';
 import { BANKLESS } from '../constants/Bankless';
 
+
 Router.events.on('routeChangeStart', () => NProgress.start());
 Router.events.on('routeChangeComplete', () => NProgress.done());
 Router.events.on('routeChangeError', () => NProgress.done());
 
+/* ---------Start Sentry.io installation--------- */
```

Comment:
Recommend the logic be moved to a separate file, then imported into the app.tsx file